tests/wake-format/basic/stdout

```diff
@@ -708,9 +741,14 @@ def loop aaaaa sssss eeeee =
 # A dictionary associating a key-value pair, providing fast lookup by key.
 # This is internally implemented by a balanced tree, so some total ordering must
 # be able to be produced for the key type.
+
+
 global export tuple Map k v =
     # The ordering function in use, over the key type only.  The `Tree` also
+    
+    
     # stores a version of this over the full `Pair` type, but some of the data
+    
```

Comment:
> how folding a floating-comment check into consume_wsnlc and similar won't be sufficient

So we could have `consume_wsnlc()` _stop_ at a floating comment but then any time we have a `consume_wsnlc()` we'd have to add a condition for "are we at a floating comment or the thing we expected.

Alternatively it might be possible to have `consume_wsnlc()` re-emit the floating comment but I'm not sure that it would have enough information to do so.

Mainly though I think my stance is that non-top-level floating comment feel fairly uncommon and less likely to be messed up so it doesn't feel worth the code/API complications

Reply:
Makes sense, thanks!